dist/src/main/java/io/camunda/application/commons/backup/BackupPriorityConfiguration.java

```diff
@@ -65,20 +73,23 @@
 import org.springframework.core.env.Environment;
 
 @Configuration
-@ProfileOperateTasklist
+@ProfileWebApp
 public class BackupPriorityConfiguration {
 
   private static final Logger LOG = LoggerFactory.getLogger(BackupPriorityConfiguration.class);
   private static final String NO_CONFIG_ERROR_MESSAGE =
-      "Expected operate or tasklist to be configured, but none of them is.";
+      "Expected operate, tasklist or optimize to be configured, but none of them is.";
 
   final OperateProperties operateProperties;
   final TasklistProperties tasklistProperties;
   final String[] profiles;
+  private final String optimizePrefix;
+  private final Boolean optimizeIsElasticSearch;
 
   public BackupPriorityConfiguration(
       @Autowired(required = false) final OperateProperties operateProperties,
       @Autowired(required = false) final TasklistProperties tasklistProperties,
+      @Autowired(required = false) final OptimizeIndexNameService optimizeIndexNameService,
```

Comment:
@RomanJRW  what should we do in case this is not autowired? Is it a possibility?